microsoft-edge/webview2/code-samples-links.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Links to WebView2 sample code hosted on GitHub.
+title: WebView2 sample code
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 10/18/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, source code, sample code, samples, code samples, WebView sample code, WebView2 sample code
+---
+# WebView2 sample code
+
+The following WebView2 samples are available in the Get Started guides for WebView2 and in the [WebView2 Samples repo](https://github.com/MicrosoftEdge/WebView2Samples).
+
+
+<!-- ====================================================================== -->
+## Sample code for Get Started guides
+
+The WebView2 "Get started" guides use starter code for Win32, Windows Presentation Foundation (WPF), WinForms, and WinUI.
+
+| Task | Article | Code |
+|---|---|---|
+| Create a WebView2 in a Win32 application | [Get started with WebView2 in Win32 apps](./get-started/win32.md) | Starter code at [WebView2 Samples Win32 Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/Win32_GettingStarted) |
+| Set up a WebView2 in a WPF application | [Get started with WebView2 in WPF apps](./get-started/wpf.md) | .NET starter code at [WebView2 Samples WPF Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/WPF_GettingStarted) |
+| Create a WebView2 in a WinForms application | [Get started with WebView2 in WinForms apps](./get-started/winforms.md) | .NET starter code at [WebView2 Samples WinForms Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/WinForms_GettingStarted) |
+| Set up a WebView2 in a WinUI 3 application | [Get started with WebView2 in WinUI 3 (Windows App SDK)](./get-started/winui.md) | Starter code at [WebView2 Samples WinUI Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/WinUI3_GettingStarted) |
+| Create a WebView2 in WinUI 2 application | [Get started with WebView2 in WinUI 2 apps guide](./get-started/winui2.md) | Uses the WinUI 2 NuGet package.  Has no sample code. |
+
+
+<!-- ====================================================================== -->
+## WebView2 sample apps
+
+WebView2 sample apps include solutions for Win32, WPF, WinForms, and WinUI.  The Microsoft Visual Studio solution file [WebView2Samples.sln](https://github.com/MicrosoftEdge/WebView2Samples/blob/master/SampleApps/WebView2Samples.sln) gathers the following samples.
+
+When you load `WebView2Samples.sln` into Visual Studio, the following projects appear in Solution Explorer, as folders:
+
```

Comment:
@vchapel - Needs clearer descriptions in the Code column cells.  Try copying from each project's readme at the code repo.  eg what is the "WebView2SampleWinComp" sample for, compared to the "WebView2APISample" sample?